Add `roles` api for `AudioTrack` and `SubtitleTrack` (#640)
bitmovin/bitmovin-player-react-native: CHANGELOG.md

- Update Bitmovin's native Android SDK version to `3.109.0`
- Update Bitmovin's native iOS SDK version to `3.90.0`

### Added

> - `SubtitleTrack.roles` and `AudioTrack.roles` to expose list of associated `MediaTrackRole` information

Reviewer: nit

```suggestion
- `SubtitleTrack.roles` and `AudioTrack.roles` to list the associated `MediaTrackRole` information
```
